### ISSUE TITLE
Captions and transcripts font letter ğ

### ISSUE THREAD

**Issue by pollecuttn** [wellcomecollection/wellcomecollection.org#8482]
**What**

C’s&T’s The title “Nazar boncuğu” is presenting strangely with the font on the g

**Where**

https://preview.wellcomecollection.org/guides/exhibitions/YyGb2xEAAHzJwTh0/captions-and-transcripts
You will need the exhibition guide toggle turned on to view it

**Comment by gestchild** [comment]
The issue here is that the font we're using to display these titles (Wellcome Bold) doesn't have this particular glyph.

I've just been discussing this issue with @davidpmccormick and he suggested swapping the font to Inter, as we do for Collection titles where we have experienced similar issues. He also pointed out Gareth O was moving to use Inter for more and more titles, saving Wellcome Bold for story and card titles only, so this would be consistent with that.

@DominiqueMarshall what do you think?

**Comment by DominiqueMarshall** [comment]
Yeah I'm happy to swap those titles to Inter, and would like to keep the Wellcome Bold for the main section titles (In Plain Sight, Room 1 etc)
